certbot/main.py

```diff
@@ -866,6 +867,56 @@ def _post_logging_setup(config, plugins, cli_args):
     logger.debug("Discovered plugins: %r", plugins)
 
 
+def acquire_file_lock(lock_path):
```

Comment:
Not a huge issue. I was just watching the fact that main.py is getting pretty large...